docs/blog/posts/alignment-check.md

```diff
@@ -39,14 +39,8 @@ may be reasons - from data governance and privacy rules to costs - for not sendi
 to OpenAI. One option is to use your own
 [locally hosted LLM]("../../components/self_hosted_llms/index.md").
 
-The other option is to use [AlignScore](../../components/align-score/index.md), a model
```

Comment:
DEPRECATED would mean it still works. If it's not working I don't think we should include in docs.

Reply:
Hmm fair that's the wrong word. Let's add [No longer available] at the start instead then?

Reply:
To be honest with you I don't see the value of mentioning it, especially since we want to get rid of the alignscore logic all together. Because keeping that section means keeping the docs which don't make sense if the code is deleted. And that point goes against your point of getting rid of `ALIGN_SCORE_METHOD`.
The way I see it there are two possibilities:
- Disabling alignScore: Meaning we keep the documentation but return an error when we pick `AlignScore` instead of `LLM`. 
- Deleting alignScore: We get rid of alignScore totally (including docs)

I would personally vote for 1, although both approaches are equally fine.

Reply:
let's delete it, otherwise the code could get out of date and annoying (it probably already is). The only reason I'm saying we should keep the mention here is that this is a blog post and not the main docs itself - I see this as a nice historical reference. Up to you though in the end

Reply:
let's delete it, otherwise the code could get out of date and annoying (it probably already is). The only reason I'm saying we should keep the mention here is that this is a blog post and not the main docs itself - I see this as a nice historical reference. Up to you though in the end